Uplift/Views/FacilitiesDropdownHeaderView.swift

```diff
@@ -61,21 +63,19 @@ class FacilitiesDropdownHeaderView: DropdownHeaderView {
         let todaysDate = Date()
         let dayOfWeek = todaysDate.getIntegerDayOfWeekToday()
         for detail in facility.details where !detail.times.isEmpty {
-            for time in detail.times {
-                if time.dayOfWeek == dayOfWeek {
-                    for timeRange in time.timeRanges {
-                        if todaysDate > timeRange.openTime && todaysDate < timeRange.closeTime {
-                            return true
-                        } else {
-                            return false
-                        }
+            for time in detail.times where time.dayOfWeek == dayOfWeek {
+                for timeRange in time.timeRanges {
+                    if todaysDate > timeRange.openTime && todaysDate < timeRange.closeTime {
+                        return true
+                    } else {
+                        return false
```

Comment:
could do `return todaysDate > timeRange.openTime && todaysDate < timeRange.closeTime` since the else case is `return false`

Reply:
aa too late lol

Reply:
good catch